x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -197,14 +197,11 @@ export const AlertsTableComponent: FC<DetectionEngineAlertTableProps> = ({
     [isEventRenderedView]
   );
 
-  const rowHeightsOptions: EuiDataGridRowHeightsOptions | undefined = useMemo(() => {
-    if (isEventRenderedView) {
-      return {
-        defaultHeight: 'auto',
-      };
-    }
-    return undefined;
-  }, [isEventRenderedView]);
+  const rowHeightsOptions: EuiDataGridRowHeightsOptions = useMemo(() => {
+    return {
+      defaultHeight: 'auto',
```

Comment:
@elastic/security-threat-hunting-investigations Let me know if this is the right way to increase row height to fit avatars by default.

Reply:
It looks like EUI team updated grid component and now table's row does not cut off avatar anymore. Reverted my related changes.